### PR TITLE
Fix download locations

### PR DESCRIPTION
**Pull request by psiinon**
As per #976

### CODE REVIEW

File: build/build.xml [hunk 1]
```diff
@@ -386,6 +394,7 @@
 		<build-addon name="cmss" />
 		<build-addon name="codedx"/>
 		<build-addon name="communityScripts"/>
+		<build-addon name="cspscanner"/>
```
Comment (kingthorin) [inline build/build.xml]: Did you mean to include this?

Comment (psiinon) [inline build/build.xml]: I did, sorry, I should have said - the lack of this line means that the cspscanner doesnt get included in the weekly build, even though it sound be (see the deploy-weekly target)

Comment (kingthorin) [inline build/build.xml]: Ok so this fixes something missed in https://github.com/zaproxy/zap-extensions/pull/968?

I'm fine with it being included here, just want to make sure it's doc'd somehow..

Comment (psiinon) [inline build/build.xml]: Thats right and yes, it should be documented :)

**Pull request merged**